net-misc/nextcloud-client/nextcloud-client-3.14.2.ebuild

```diff
@@ -0,0 +1,118 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake virtualx xdg
+
+DESCRIPTION="Desktop Syncing Client for Nextcloud"
+HOMEPAGE="https://github.com/nextcloud/desktop"
+SRC_URI="
+	https://github.com/nextcloud/desktop/archive/v${PV/_/-}.tar.gz -> ${P}.tar.gz
+"
+S="${WORKDIR}/desktop-${PV/_/-}"
+
+LICENSE="CC-BY-3.0 GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~ppc64 ~x86"
+IUSE="doc dolphin nautilus test webengine"
+RESTRICT="!test? ( test )"
+
+# slot op for qtqui as this package uses private API parts of qtqui
+# src/gui/generalsettings.cpp:#include <private/qzipwriter_p.h>
+RDEPEND="
+	>=dev-db/sqlite-3.34:3
+	>=dev-libs/openssl-1.1.0:0=
+	dev-libs/glib:2
+	dev-libs/qtkeychain:=[qt6(+)]
+	dev-qt/qt5compat:6
+	dev-qt/qtbase:6[dbus,gui,network,sql,widgets]
+	dev-qt/qtdeclarative:6[widgets]
+	dev-qt/qtsvg:6
+	dev-qt/qtwebsockets:6
+	net-libs/libcloudproviders
+	kde-frameworks/karchive:6
+	kde-frameworks/kguiaddons:6
+	sys-libs/zlib
+	dolphin? (
+		kde-frameworks/kcoreaddons:6
```

Comment:
Just to make sure, there was no minimum KF6 version required by build system?

Reply:
It seems to require KF6 5.240. Not sure how to map this to Gentoo versions?

https://github.com/nextcloud/desktop/blob/v3.14.2/shell_integration/CMakeLists.txt#L15

https://github.com/nextcloud/desktop/blob/v3.14.2/shell_integration/dolphin/CMakeLists.txt#L8

Reply:
That's KF6 development phase version, just use `6.0.0` then.